tests/rptest/tests/connection_rate_limit_test.py

```diff
@@ -27,8 +27,8 @@ class ConnectionRateLimitTest(RedpandaTest):
     MSG_SIZE = 1000000
     PRODUCE_COUNT = 10
     READ_COUNT = 10
-    RANDOM_READ_PARALLEL = 2
-    MAX_RETRY_COUNT = 4
+    RANDOM_READ_PARALLEL = 3
```

Comment:
Change to RANDOM_READ_PARALLEL looks unrelated to the rest -- why does increasing this help?

Reply:
I only want to increase number of connections to more fair test